src/backend/opencl/math.hpp

```diff
@@ -143,9 +143,9 @@ cfloat operator+(cfloat a, cfloat b);
 cfloat operator+(cfloat a);
 cdouble operator+(cdouble a, cdouble b);
 cdouble operator+(cdouble a);
-cfloat operator*(cfloat a, cfloat b);
-cdouble operator*(cdouble a, cdouble b);
-common::half operator+(common::half lhs, common::half rhs) noexcept;
+cfloat operator*(cfloat lhs, cfloat rhs);
+cdouble operator*(cdouble lhs, cdouble rhs);
+common::half operator+(common::half a, common::half b) noexcept;
```

Comment:
a,b to lhs,rhs for earlier two and reverse for the later ?

Reply:
clang-tidy did that. I will go back and make it consistent